cms/templates/export.html

```diff
@@ -6,18 +6,25 @@
   from django.utils.translation import ugettext as _
   import json
 %>
-<%block name="title">${_("Course Export")}</%block>
+<%block name="title">
+%if library:
```

Comment:
Nit: Here and below, I think I would define translated variables and avoid duplicating the strings when they are identical in the two cases, except for the noun.

For instance, something like
title_case_export_item = "Library"
export_item = "library"
plural_export_item = "libraries"

@sarina Do you have a preference from the perspective of the translators? The duplication might be easier for the translators to understand and translate.